doc/lua_api.md

```diff
@@ -3535,10 +3535,12 @@ Markup language used in `hypertext[]` elements uses tags that look like HTML tag
 The markup language is currently unstable and subject to change. Use with caution.
 Some tags can enclose text, they open with `<tagname>` and close with `</tagname>`.
 Tags can have attributes, in that case, attributes are in the opening tag in
-form of a key/value separated with equal signs. Attribute values should not be quoted.
+form of a key/value separated with equal signs.
+Attribute values should be quoted using either " or '.
 
 If you want to insert a literal greater-than sign or a backslash into the text,
-you must escape it by preceding it with a backslash.
+you must escape it by preceding it with a backslash. In a quoted attribute value, you
+can insert a literal quote mark by preceding it with a backslash.
```

Comment:
It seems like the `>` does not need to be escaped when it is quoted.

Reply:
That's correct. I don't really want to document all the cases where greater than can be inserted literally, it's best to just escape any text